test/testAnalysisTools.jl

```diff
@@ -0,0 +1,31 @@
+# Test for tree-based analysis tools found in `AnalysisTools.jl`.
+using Test
+using IncrementalInference
+using RoME # Used for loading canonical graphs (is this ok?).
+
+@testset "Number of non-zero calculation for frontals." begin
```

Comment:
Hi @tonioteran , just caught this now -- to engage these tests, they  must be included in IncrementalInference/test/runtests.jl.  Its a simple `include(testAnalysisTools.jl)`.  I'll add it to show.

Reply:
see a7746a2